### PR TITLE
Fix bug with LegalizeLayoutTranform which added duplicate ops

### PR DESCRIPTION
**Pull request by trevor-m**
LegalizeLayoutTransform visitor was not replacing layout_transform with transpose properly.

### CODE REVIEW

**Comment by trevor-m** [comment]
@anijain2305

File: python/tvm/relay/tensorrt.py [hunk 1]
```diff
@@ -186,6 +185,7 @@ def EnableTrt(mod, params=None, trt_version=None):
     mod = relay.transform.RemoveUnusedFunctions()(mod)
     mod = relay.transform.InferType()(mod)
     mod = relay.transform.ConvertLayout('NCHW')(mod)
+    print(mod['main'])
```
Comment (anijain2305) [inline python/tvm/relay/tensorrt.py]: Let's remove the print?

Comment (trevor-m) [reply python/tvm/relay/tensorrt.py]: Oops, fixed.

**Review by anijain2305 — APPROVED** [review]
LGTM

**Pull request merged**